build.gradle

```diff
@@ -121,4 +139,10 @@ bootJar {
 
 jar {
     enabled = true
+    manifest {
+        attributes(
+            'Class-Path': configurations.compile.collect { it.getName() }.join(' '),
+            'Main-Class': 'Main'
```

Comment:
I don't think we need main class, it's going to be a library jar.

Reply:
It's provided for testing